docs/developers/running-locally.md

```diff
@@ -1,4 +1,4 @@
-# Development Environment setup
```

Comment:
Does the README need an update for this file name change?

Reply:
renamed file and updated README.md